sourcemash/frontend/static/js/routers/app_router.js

```diff
@@ -26,6 +27,15 @@ Sourcemash.Routers.AppRouter = Backbone.Router.extend({
         this._swapView(feedsView);
     },
 
+    browseFeeds: function() {
+        var browseView = new Sourcemash.Views.BrowseView({
+            collection: new Sourcemash.Collections.Feeds(),
```

Comment:
Add an option to the feeds collection for {all: true/false}, so then in the url function in the Feeds Collection file, we can add all 'if options.all.'